file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [18-Apr-2025 06:55:05 UTC] PHP Parse error:  syntax error, unexpected '_000_000' (T_STRING), expecting ';' in /wordpress/wp-content/plugins/zaprite-payment-gateway/includes/utils.php on line 14
[18-Apr-2025 06:55:14 UTC] PHP Parse error:  syntax error, unexpected '_000_000' (T_STRING), expecting ';' in /wordpress/wp-content/plugins/zaprite-payment-gateway/includes/utils.php on line 14
[18-Apr-2025 06:55:22 UTC] PHP Parse error:  syntax error, unexpected '_000_000' (T_STRING), expecting ';' in /wordpress/wp-content/plugins/zaprite-payment-gateway/includes/utils.php on line 14
[18-Apr-2025 06:55:31 UTC] PHP Deprecated:  Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6121
[18-Apr-2025 06:55:31 UTC] PHP Deprecated:  Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6121
[18-Apr-2025 06:55:41 UTC] PHP Deprecated:  Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6121
[18-Apr-2025 06:55:41 UTC] PHP Deprecated:  Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6121
[18-Apr-2025 06:55:49 UTC] PHP Deprecated:  Function register_sidebar_widget is <strong>deprecated</strong> since version 2.8.0! Use wp_register_sidebar_widget() instead. in /wordpress/wp-includes/functions.php on line 6121
[18-Apr-2025 06:55:49 UTC] PHP Deprecated:  Function register_widget_control is <strong>deprecated</strong> since version 2.8.0! Use wp_register_widget_control() instead. in /wordpress/wp-includes/functions.php on line 6121
[18-Apr-2025 06:57:42 UTC] PHP Deprecated:  Required parameter $option follows optional parameter $number in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 42
[18-Apr-2025 06:57:42 UTC] PHP Warning:  Undefined array key "page" in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 24
[18-Apr-2025 06:57:43 UTC] PHP Deprecated:  Required parameter $option follows optional parameter $number in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 42
[18-Apr-2025 06:57:43 UTC] PHP Warning:  Undefined array key "page" in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 24
[18-Apr-2025 06:57:50 UTC] PHP Deprecated:  Required parameter $option follows optional parameter $number in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 42
[18-Apr-2025 06:57:50 UTC] PHP Warning:  Undefined array key "page" in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 24
[18-Apr-2025 06:57:51 UTC] PHP Deprecated:  Required parameter $option follows optional parameter $number in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 42
[18-Apr-2025 06:57:51 UTC] PHP Warning:  Undefined array key "page" in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 24
[18-Apr-2025 06:57:59 UTC] PHP Deprecated:  Required parameter $option follows optional parameter $number in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 42
[18-Apr-2025 06:57:59 UTC] PHP Warning:  Undefined array key "page" in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 24
[18-Apr-2025 06:58:00 UTC] PHP Deprecated:  Required parameter $option follows optional parameter $number in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 42
[18-Apr-2025 06:58:00 UTC] PHP Warning:  Undefined array key "page" in /wordpress/wp-content/plugins/zenphoto-shorttags/zenphoto-shorttags.php on line 24
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_zefw_custom_emails (
	        id bigint(20) NOT NULL AUTO_INCREMENT,
	        json JSON DEFAULT NULL,
	        triggers JSON DEFAULT NULL,
	        status varchar(32),
	        css LONGTEXT,
	        email_type VARCHAR(256),
	        template BOOLEAN,
	       	save_time bigint(20),
			user_ID bigint(20),
	        primary key (id)
	    ) DEFAULT CHARACTER SET utf8mb4</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: MySQL field type cannot be translated to SQLite: json.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')
#2 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(24): dbDelta(Array)
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(33): zefw_create_custom_emails_table()
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query CREATE TABLE wp_zefw_custom_emails (
	        id bigint(20) NOT NULL AUTO_INCREMENT,
	        json JSON DEFAULT NULL,
	        triggers JSON DEFAULT NULL,
	        status varchar(32),
	        css LONGTEXT,
	        email_type VARCHAR(256),
	        template BOOLEAN,
	       	save_time bigint(20),
			user_ID bigint(20),
	        primary key (id)
	    ) DEFAULT CHARACTER SET utf8mb4 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, zefw_create_custom_emails_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT id FROM wp_zefw_custom_emails</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT id FROM wp_zefw_custom_emails | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT id FROM ...')
#2 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(45): wpdb->get_results('SELECT id FROM ...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#10 {main}</pre>
 for query SELECT id FROM wp_zefw_custom_emails made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_cancelled_order', '0', '1744959552', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, admin_cancelled_order, 0, 1744959552, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_cancelled_order', '0', '1744959552', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_failed_order', '0', '1744959552', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, admin_failed_order, 0, 1744959552, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_failed_order', '0', '1744959552', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_new_order', '0', '1744959552', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, admin_new_order, 0, 1744959552, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_new_order', '0', '1744959552', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'completed_order', '0', '1744959552', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, completed_order, 0, 1744959552, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'completed_order', '0', '1744959552', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'customer_note', '0', '1744959552', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, customer_note, 0, 1744959552, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'customer_note', '0', '1744959552', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'invoice', '0', '1744959552', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, invoice, 0, 1744959552, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'invoice', '0', '1744959552', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'new_account', '0', '1744959552', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, new_account, 0, 1744959552, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'new_account', '0', '1744959552', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'order_on_hold', '0', '1744959552', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, order_on_hold, 0, 1744959552, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'order_on_hold', '0', '1744959552', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'processing_order', '0', '1744959552', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, processing_order, 0, 1744959552, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'processing_order', '0', '1744959552', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'refunded_order', '0', '1744959552', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, refunded_order, 0, 1744959552, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'refunded_order', '0', '1744959552', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'reset_password', '0', '1744959552', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, reset_password, 0, 1744959552, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'reset_password', '0', '1744959552', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:21 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE wp_zefw_custom_emails (
	        id bigint(20) NOT NULL AUTO_INCREMENT,
	        json JSON DEFAULT NULL,
	        triggers JSON DEFAULT NULL,
	        status varchar(32),
	        css LONGTEXT,
	        email_type VARCHAR(256),
	        template BOOLEAN,
	       	save_time bigint(20),
			user_ID bigint(20),
	        primary key (id)
	    ) DEFAULT CHARACTER SET utf8mb4</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: MySQL field type cannot be translated to SQLite: json.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')
#2 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(24): dbDelta(Array)
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(33): zefw_create_custom_emails_table()
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query CREATE TABLE wp_zefw_custom_emails (
	        id bigint(20) NOT NULL AUTO_INCREMENT,
	        json JSON DEFAULT NULL,
	        triggers JSON DEFAULT NULL,
	        status varchar(32),
	        css LONGTEXT,
	        email_type VARCHAR(256),
	        template BOOLEAN,
	       	save_time bigint(20),
			user_ID bigint(20),
	        primary key (id)
	    ) DEFAULT CHARACTER SET utf8mb4 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, zefw_create_custom_emails_table, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:21 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT id FROM wp_zefw_custom_emails</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT id FROM wp_zefw_custom_emails | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT id FROM ...')
#2 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(45): wpdb->get_results('SELECT id FROM ...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#10 {main}</pre>
 for query SELECT id FROM wp_zefw_custom_emails made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:21 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_cancelled_order', '0', '1744959561', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, admin_cancelled_order, 0, 1744959561, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_cancelled_order', '0', '1744959561', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:21 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_failed_order', '0', '1744959561', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, admin_failed_order, 0, 1744959561, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_failed_order', '0', '1744959561', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_new_order', '0', '1744959561', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, admin_new_order, 0, 1744959561, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'admin_new_order', '0', '1744959561', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'completed_order', '0', '1744959561', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, completed_order, 0, 1744959561, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'completed_order', '0', '1744959561', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'customer_note', '0', '1744959561', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, customer_note, 0, 1744959561, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'customer_note', '0', '1744959561', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'invoice', '0', '1744959561', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, invoice, 0, 1744959561, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'invoice', '0', '1744959561', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'new_account', '0', '1744959561', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, new_account, 0, 1744959561, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'new_account', '0', '1744959561', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'order_on_hold', '0', '1744959561', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, order_on_hold, 0, 1744959561, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'order_on_hold', '0', '1744959561', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'processing_order', '0', '1744959561', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, processing_order, 0, 1744959561, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'processing_order', '0', '1744959561', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'refunded_order', '0', '1744959561', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, refunded_order, 0, 1744959561, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'refunded_order', '0', '1744959561', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'reset_password', '0', '1744959561', '0')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES (:param0 , NULL, :param1 , :param2 , :param3 , :param4 ) | parameters: disabled, reset_password, 0, 1744959561, 0</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_zefw_custom_emails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_zefw_custom_...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/zesty-emails-custom-template-designer-for-woocommerce/init/tables.php(55): wpdb->insert('wp_zefw_custom_...', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): zefw_update_db_tracking_table_check('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(578): do_action('plugins_loaded')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_zefw_custom_emails` (`status`, `css`, `email_type`, `template`, `save_time`, `user_ID`) VALUES ('disabled', NULL, 'reset_password', '0', '1744959561', '0') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('plugins_loaded'), WP_Hook->do_action, WP_Hook->apply_filters, zefw_update_db_tracking_table_check, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-Apr-2025 06:59:57 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method zia3::flush() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_zia3-c...', false)
#3 /internal/eval.php(20): activate_plugin('zia3-css-fullsc...')
#4 {main}
  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324
[18-Apr-2025 07:00:05 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method zia3::flush() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_zia3-c...', false)
#3 /internal/eval.php(20): activate_plugin('zia3-css-fullsc...')
#4 {main}
  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324
[18-Apr-2025 07:00:14 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method zia3::flush() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_zia3-c...', false)
#3 /internal/eval.php(20): activate_plugin('zia3-css-fullsc...')
#4 {main}
  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #1 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"zia3-css-fullscreen-background-slideshow"},"options":{"activate":true}}) : PHP.run() failed with exit code 255 and the following output: 
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      ... 2 lines matching cause stack trace ...
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: 
        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)
        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)
        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)
        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)
        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
      response: PHPResponse {
        httpStatusCode: 500,
        headers: {
          'x-powered-by': [ 'PHP/8.0.30-dev' ],
          'content-type': [ 'text/html; charset=UTF-8' ]
        },
        bytes: Uint8Array(0) [],
        exitCode: 255,
        errors: ''
      },
      source: 'request'
    }
  }
}

Node.js v23.11.0
